src/__tests__/ApiContainer.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/Store';
+import Apicontainer from '../components/ApiContainer';
+
```

Comment:
- [ ] Good job adding the integration test, however, also ensure to add unit tests for some pure functions in Redux code.